packages/contracts/src/deploy-config.ts

```diff
@@ -129,6 +129,11 @@ interface RequiredDeployConfig {
    */
   finalizationPeriodSeconds: number
 
+  /**
+   * The period seconds for which challenges can be created per each output.
+   */
+  colosseumCreationPeriod: number
```

Comment:
You're right.
The differences in range between uint64 and js number type could potentially be a problem.
However, since the actual values being used are within the range of js number, it's still acceptable for now.